eppo_core/src/eval/eval_precomputed.rs

```diff
@@ -76,6 +76,18 @@ pub fn get_precomputed_configuration(
                         return None;
                     }
 
+                    let flag_actions = if let Some(actions) = actions.get(flag_key) {
```

Comment:
leaking in unintended changes

Reply:
whoa that is strange.. must have been from the merge conflict. sorry about that, let me figure out what happen